metadata-ingestion/src/datahub/ingestion/source/powerbi/m_query/pattern_handler.py

```diff
@@ -959,28 +961,50 @@ def create_lineage(
         )
 
         if not connect_string:
-            logger.warning(
-                "Can not extract ODBC connect string from data access function"
+            self.reporter.warning(
+                title="Can not extract ODBC connect string",
+                message="Can not extract ODBC connect string from data access function. Skipping Lineage creation.",
+                context=f"table-name={self.table.full_name}",
```

Comment:
```suggestion
                context=f"table-name={self.table.full_name}, data-access-func-detail={data_access_func_detail}",
```

this may be useful to quickly spot the issue

Reply:
Added